### ISSUE TITLE
Add storage tests

### ISSUE THREAD

**Issue by kostko** [oasisprotocol/oasis-sdk#20]
There should be a set of tests for the high-level storage helpers.

**Comment by pro-wh** [comment]
can we say #37 added this?

**Comment by kostko** [comment]
No, I was thinking more about unit tests for the code in `runtime-sdk/src/storage`.
